src/components/children/NestedInfoTable.jsx

```diff
@@ -24,150 +24,161 @@ class NestedInfoTable extends React.Component {
+											size="small"
+											className={classes.inlineButtons}
+											onClick={() => onClick(data[key].datasetId)}
+										>
+											Preview
+										</Button>
+										{data[key].datasetId}
+									</ListItem>
+									<Divider />
+								</List>
+							</TableCell>
+						</TableRow>
+					);
+				} else if (key === "datasetId" && data[key]) {
+					return (
+						<TableRow key={key}>
+							<TableCell className={classes.rowHeaderCell}>{key}</TableCell>
+							<TableCell>
+								<List>
+									<ListItem key={data[key]}>
+										<Button
+											color="primary"
+											variant="contained"
+											size="small"
+											className={classes.inlineButtons}
+											onClick={() => onClick(data[key])}
+										>
+											Preview
+										</Button>
+										{data[key]}
+									</ListItem>
+									<Divider />
+								</List>
+							</TableCell>
+						</TableRow>
+					);
+				} else if (data[key]) {
+					return (
+						<TableRow key={key}>
+							{key === "creator" ? (
+								<Tooltip title="The person who created the resource">
+									<TableCell className={classes.rowHeaderCell}>{key}</TableCell>
+								</Tooltip>
+							) : (
+								<TableCell className={classes.rowHeaderCell}>{key}</TableCell>
+							)}
+							{/* If the value is an object, render a sub-table	*/}
+							{typeof data[key] === "object" && data[key] ? (
+								Object.keys(data[key]).map((key2) => {
+									return (
+										<TableRow key={key2}>
+											<TableCell className={classes.rowSubHeaderCell}>{key2}</TableCell>
 
-																	);
-																}
-															}) : (
-																<TableCell>{JSON.stringify(this.props.data[key][key2])}</TableCell>)
+											{typeof data[key][key2] === "object" && data[key][key2] ? (
+												Object.keys(data[key][key2]).map((key3) => {
+													if (key3 !== "legacyEntry") {
+														return (
+															<TableRow key={key3}>
+																<TableCell className={classes.rowSubHeaderCell}>
+																	{key3}
+																</TableCell>
+																<TableCell>
+																	{JSON.stringify(data[key][key2][key3])}
+																</TableCell>
+															</TableRow>
+														);
 													}
-												</TableRow>
-											);
-										})
-										:
-										(<TableCell>{this.props.data[key]}</TableCell>)
-									}
-								</TableRow>);
-						} else {
-							return null;
-						}
-					})
+												})
+											) : (
+												<TableCell>{JSON.stringify(data[key][key2])}</TableCell>
+											)}
+										</TableRow>
+									);
+								})
+							) : key === "dataType" && startsWithErgoOrIncore(data[key]) ? ( // Another ternary operator to check if the value is a dataType if so create linking
+								<TableCell>
+									<a
+										href={`${config.semanticServiceType}${data[key]}`}
+										target="_blank"
+										rel="noopener noreferrer"
```

Comment:
Maybe try the theme color #e8a114
<img width="485" alt="image" src="https://github.com/IN-CORE/incore-ui/assets/13950475/ed7cb5b7-5157-4bf1-a214-9ca0a216647c">
What do you think? up to you.